tests/integration/workflow/dummy.py

```diff
@@ -43,6 +44,26 @@ class DummyInference(Inference):
     score: float
 
 
+class LocalConfig:
+    extra = Extra.allow
```

Comment:
Should we add extra config to every existing dataclass (test samples/annotations/assets)?